docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc

```diff
@@ -4239,23 +4239,44 @@ For the searched DELETE form, if no WHERE clause exists, all rows are deleted fr
 ```
 Searched DELETE is:
 
-DELETE FROM table
-
-    [WHERE search-condition ]
-       [[for] access-option access]
+DELETE 
+[WITH NO ROLLBACK] ["["FIRST N"]"]
+FROM table-name
+[WHERE search-condition ]
+[[for] access-option access]
 
 access-option is:
-    read committed
+read committed
 ```
 
 [[delete_syntax]]
 === Syntax Description of DELETE
 
-* `_table_`
+* `_table-name_`
 +
 names the user table or view from which to delete rows. _table_ must be a base table or an updatable view. To refer to a
 table or view, use the ANSI logical name. See <<database_object_names,Database Object Names>>.
 
+* `WITHOUT NO ROLLBACK`

```

Comment:
This should be "WITH NO ROLLBACK"

Reply:
Oops  :stuck_out_tongue_closed_eyes: thanks Dave, I've fixes that :blush: